subsys/bluetooth/controller/ll_sw/ull_llcp_local.c

```diff
@@ -382,13 +382,19 @@ static void lr_st_disconnect(struct ll_conn *conn, uint8_t evt, void *param)
 static void lr_st_idle(struct ll_conn *conn, uint8_t evt, void *param)
 {
 	struct proc_ctx *ctx;
+	enum llcp_proc curr_proc;
 
 	switch (evt) {
 	case LR_EVT_RUN:
 		ctx = llcp_lr_peek(conn);
 		if (ctx) {
+			/*
+			 * since the call to lr_act_run may release the context we need to remember
+			 * which procedure we are running
+			 */
+			curr_proc = ctx->proc;
```

Comment:
```suggestion
			const enum llcp_proc curr_proc = ctx->proc;

```

Reply:
done

Reply:
I do not see the `const`, did you forget to push?